packages/cardano-services/test/Metadata/HttpMetadataService.test.ts

```diff
@@ -0,0 +1,172 @@
+/* eslint-disable max-len */
+import { Cardano, ProviderError, ProviderFailure } from '@cardano-sdk/core';
+import { ExtMetadataFormat, createHttpMetadataService } from '../../src/Metadata';
+import { adaPoolsExtMetadataMock, cip6ExtMetadataMock, mainExtMetadataMock } from './mocks';
+import { createGenericMockServer } from '@cardano-sdk/util-dev';
+import { dummyLogger } from 'ts-log';
+import url from 'url';
+
+export const mockPoolExtMetadataServer = createGenericMockServer((handler) => async (req, res) => {
+  const result = handler(req);
+
+  res.setHeader('Content-Type', 'application/json');
+
+  if (result.body) {
+    res.statusCode = result.code || 200;
+    return res.end(JSON.stringify(result.body));
+  }
+
+  const reqUrl = url.parse(req.url!).pathname;
+
+  if (reqUrl === `/${ExtMetadataFormat.AdaPools}`) {
+    return res.end(JSON.stringify(adaPoolsExtMetadataMock));
+  } else if (reqUrl === `/${ExtMetadataFormat.CIP6}`) {
+    return res.end(JSON.stringify(cip6ExtMetadataMock));
+  }
+
+  return res.end(JSON.stringify(adaPoolsExtMetadataMock));
+});
+
+describe('HttpMetadataService', () => {
+  describe('healthy state', () => {
```

Comment:
Can you add a test to cover the scenario of metadata including both extended properties? In this case, CIP6 should take priority.

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/490/commits/eb46deb268607fe1f476d98dd6e72ce3e16caf7a